### DISCUSSION TITLE
Disable sidebar menu items

### DISCUSSION THREAD

**Discussion by pellyadolfo** [facebook/docusaurus#10543] [Q&A]
Hello,

I am looking for a way to disable sidebar menu items, either categories or docs. Is this possible / desirable? 

I could not find any related topic and it looks to me a quite basic version to be missed and easy to implement.

Thanks

**Comment by slorber** [comment]
What does it mean to disable sidebar menu items? 

Please share a mockup or explain the expected visual/behavior

**Reply by pellyadolfo** [reply]
Well, e.g. in the left sidebar of this page 

https://docusaurus.io/docs/category/guides 

having menu items greyed out and disabled, which means, they are there, but you cannot click on them to open a linked page.

If I check the properties in sidebar items, I would expected a 'disabled', or better 'enabled' default to true property to offer this behaviour, but it is not there.

https://docusaurus.io/docs/sidebar/items

**Reply by slorber** [reply]
> but you cannot click on them to open a linked page.

So, the page actually exists (ie users can open it by other means) and you just want the link to be greyed out and no-op? 

What's the use-case for this? Do you understand that despite you trying to disabling the link, users can still open that page? Why not simply remove that content from your sidebar in the first place?

Note: sidebar items are often links, and there's no concept of a "disabled link" in HTML. However, you can grey the link with CSS and prevent user clicks. All sidebar items accept a `className` prop that allows you to do so.

I'd recommend considering not generating the target pages if they are not meant to be accessed. You can still register custom sidebar items if you want to add "placeholders" using any HTML markup you want (and this should probably not be links `<a>` tags):

https://docusaurus.io/docs/sidebar/items#sidebar-item-html

**Reply by pellyadolfo** [reply]
yes, correct, in my use case I was considering not having yet the pages generated and offering the visitor a view of what is coming. I was wondering if there was some provision for this in existing docusaurus but certainly I can produce a similar effect with a custom markup. 

Wih this, for now you can close the ticket then. Maybe someone else in future finds a similar use. Thanks
